e2e_deployment_files/limited_container_storage_deployment.amqp.template.json

```diff
@@ -0,0 +1,439 @@
+{
+  "modulesContent": {
```

Comment:
is it possible to combine limited_container_storage_deployment.amqp.template.json with limited_container_storage_deployment.mqtt.template.json?

Reply:
yup